cocos/renderer/CCGLProgram.cpp

```diff
@@ -230,7 +220,7 @@ bool GLProgram::initWithByteArrays(const GLchar* vShaderByteArray, const GLchar*
     std::string replacedDefines = "";
     replaceDefines(compileTimeDefines, replacedDefines);
 
-    _vertShader = _fragShader = 0;
+    clearShader();
```

Comment:
Because in this change we won't delete these shaders after linked, if only clear these two mark, the shader loaded will never been deleted, it will cause memory leak.